### PR TITLE
Refractoring aka OCD

### PR DESCRIPTION
**Pull request by Samistine**

### CODE REVIEW

**Comment by bendem** [comment]
Mixed tabs and spaces, also a bunch of `.toString()` calls can be removed when logging.

Other than that, I couldn't spot any behavior changes, this is pure syntactic improvements.

**Comment by vemacs** [comment]
Refactor*

File: Essentials/src/com/earth2me/essentials/I18n.java [hunk 1]
```diff
@@ -27,10 +27,12 @@
     private final transient IEssentials ess;
     private static final Pattern NODOUBLEMARK = Pattern.compile("''");
     private static final ResourceBundle NULL_BUNDLE = new ResourceBundle() {
+		@Override
```
Comment (mbax) [inline Essentials/src/com/earth2me/essentials/I18n.java]: tabs

File: Essentials/src/com/earth2me/essentials/I18n.java [hunk 2]
```diff
@@ -27,10 +27,12 @@
...
         public Enumeration<String> getKeys() {
             return null;
         }
 
+		@Override
```
Comment (mbax) [inline Essentials/src/com/earth2me/essentials/I18n.java]: tabs

File: Essentials/src/com/earth2me/essentials/ManagedFile.java [hunk 3]
```diff
@@ -44,45 +44,33 @@ public ManagedFile(final String filename, final IEssentials ess) {
...
-                    writer.close();
-                }
-            } finally {
-                digestStream.close();
-            }
-        } finally {
-            reader.close();
-        }
+            try (DigestOutputStream digestStream = new DigestOutputStream(new FileOutputStream(file), digest); OutputStreamWriter writer = new OutputStreamWriter(digestStream)) {
+				final char[] buffer = new char[BUFFERSIZE];
```
Comment (mbax) [inline Essentials/src/com/earth2me/essentials/ManagedFile.java]: tabs

**Comment by Samistine** [comment]
Netbeans seems to ignore my settings for spacing and indents for Essentials. It only happens in this project. It also does this, also only in essentials project:

[image http://i.imgur.com/cgBPYvZ.png]

**Comment by mbax** [comment]
> also a bunch of `.toString()` calls can be removed when logging.

File: Essentials/src/com/earth2me/essentials/metrics/MetricsLite.java [hunk 4]
```diff
@@ -162,6 +162,7 @@ public boolean start() {
 
                 private boolean firstPost = true;
 
+				@Override
```
Comment (bendem) [inline Essentials/src/com/earth2me/essentials/metrics/MetricsLite.java]: tabs

**Comment by drtshock** [comment]
Still waiting on some fixes ^^

**Comment by drtshock** [comment]
https://github.com/drtshock/Essentials/pull/353#issuecomment-193417249 this could be the one if you want..

**Pull request closed without merging**